api-spec/protobuf/ark/v1/explorer.proto

```diff
@@ -44,3 +49,11 @@ message ListVtxosResponse {
   repeated Vtxo spendable_vtxos = 1;
   repeated Vtxo spent_vtxos = 2;
 }
+
+message SubscribeForAddressRequest {
+  string address = 1;
+}
+message SubscribeForAddressResponse {
```

Comment:
Should we also return the following events: 
* when a vtxo is swept ?
* when a vtxo is unilaterally put onchain ?